docs/deployment/prerequisites.md

```diff
@@ -0,0 +1,75 @@
+---
+title: UDS Prerequisites
+type: docs
+weight: 4
+---
+
+## UDS installation prerequisites
+
+`UDS Core` could run in any [Kuberentes](https://kubernetes.io/) setup, but sometimes customizations are needed based on environemnts. This is an attempt to document and link to relevant information to aid in setting up your [Kuberentes](https://kubernetes.io/) environment to ensure a successful `UDS Core` installation.  
+
+### RHEL
+---
+#### *ISTIO related changes*
+Solution is to create file `/etc/modules-load.d/istio-iptables.conf` with this content:
+
+```bash
+# These modules need to be loaded on boot so that Istio (as required by
+# UDS Core) runs properly.
+#
+# See also: https://github.com/istio/istio/issues/23009
+
+br_netfilter
+nf_nat
+xt_REDIRECT
+xt_owner
+iptable_nat
+iptable_mangle
+iptable_filter
+```
+
+```bash
+sudo systemctl stop firewalld
+sudo systemctl disable firewalld
+```
+
+### RKE2
+--- 
+* [Installation requirements](https://docs.rke2.io/install/requirements)
+* [Firewalld network conflicts](https://docs.rke2.io/known_issues#firewalld-conflicts-with-default-networking)
+* [Disabling components, such as Ingress which clashes with istio](https://docs.rke2.io/advanced#disabling-server-charts)
+* [Defense Unicorns os prep script for rke2](https://github.com/defenseunicorns/uds-rke2-image-builder/blob/main/packer/scripts/os-prep.sh)
```

Comment:
I think I would suggest teasing apart the specific pieces here per-package rather than linking this script directly, added a few specific comments for promtail and neuvector below which are the key ones. It's also more broadly applicable than just RKE2, so I'd lean towards shifting down to that core section.